src/Containers/Microsoft.NET.Build.Containers/KnownStrings.cs

```diff
@@ -31,6 +33,8 @@ public static class Properties
 
     public static class ErrorCodes
     {
+        public static readonly string CONTAINER002 = nameof(CONTAINER002);
+        public static readonly string CONTAINER003 = nameof(CONTAINER003);
```

Comment:
We have started all the codes from 10xx, is it intentional?

Reply:
The diagnostics emitted directly from the targets are still three-digits and so I just kept on with that for compat. We could break this now if we wanted.